backend/myproject/backend_app/static/javascript/login.js

```diff
@@ -44,8 +44,7 @@ function handleFormSubmit(event) {
     fetch("/api/login/", {
         method: "POST",
         headers: {
-            "Content-Type": "application/json",
-            "X-CSRFToken": csrftoken
```

Comment:
Why do you remove the token?

Reply:
because it does not add to security, since the session tokens are enough. but it interferes very often. chatty assured me that we dont need it.